nepi/main/tests/factories.py

```diff
@@ -14,6 +14,7 @@ class Meta:
     password = factory.PostGenerationMethodCall('set_password', 'test')
     last_name = factory.Sequence(lambda n: "Last%d" % n)
     first_name = factory.Sequence(lambda n: "First%d" % n)
+    email = factory.Sequence(lambda n: "username%d@localhost" % n)
```

Comment:
Curious how you figured out that this was missing. I started a 1.11 branch, did the usual bits and hit the failing tests on the mail stuff. Got distracted with other stuff before I could really dig into it, but I don't think this fix was very obvious.